### PR TITLE
Change dto sent to CreateMessage and adjust unit test

### PR DESCRIPTION
**Pull request by johevemi**
## Description

Corrects a mistake made in #106 where the wrong dto was sent to CreateMessage in exchange and exchange neighbor strategies 

## References

* #0000

### CODE REVIEW

**Review by kristianschneider — APPROVED** [review]

File: source/coordinator/GreenEnergyHub.Aggregation.Tests/Step01ExchangePerNeighbourStrategyTests.cs [hunk 1]
```diff
@@ -49,6 +49,7 @@ public void Check_Content_Of_ExchangeNeighbour_Message_Test()
             var message = (AggregatedExchangeNeighbourResultMessage)exchangeStrategy.PrepareMessages(testData, "D03", beginTime, endTime).First();
 
             // Assert
+            message.EnergyObservation.Should().HaveCount(1); // there should be only have 1 EnergyObservations in grid area 500, see ExchangeNeighbour.json
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Tests/Step01ExchangePerNeighbourStrategyTests.cs:52]: When selecting message you could filter the message for only grid area 500, that way the code isn't vulnerable to changes to the test file.

File: source/coordinator/GreenEnergyHub.Aggregation.Tests/Step02ExchangeStrategyTests.cs [hunk 2]
```diff
@@ -50,6 +50,7 @@ public void Check_Content_Of_Exchange_Message_Test()
             var message = (AggregationResultMessage)exchangeStrategy.PrepareMessages(testData.ToList(), "D03", beginTime, endTime).First();
 
             // Assert
+            message.EnergyObservation.Should().HaveCount(2); // there should be only have 2 EnergyObservations in grid area 500, see Exchange.json
```
Comment (ZavezX) [inline source/coordinator/GreenEnergyHub.Aggregation.Tests/Step02ExchangeStrategyTests.cs:53]: Same here as stated above.

**Review by ZavezX — CHANGES_REQUESTED** [review]

File: source/coordinator/GreenEnergyHub.Aggregation.Tests/Step01ExchangePerNeighbourStrategyTests.cs (hunk 1 above)
Comment (johevemi) [reply source/coordinator/GreenEnergyHub.Aggregation.Tests/Step01ExchangePerNeighbourStrategyTests.cs]: as the test is right now, we want it to fail if the test data is changed

File: source/coordinator/GreenEnergyHub.Aggregation.Tests/Step02ExchangeStrategyTests.cs (hunk 2 above)
Comment (johevemi) [reply source/coordinator/GreenEnergyHub.Aggregation.Tests/Step02ExchangeStrategyTests.cs]: as the test is right now, we want it to fail if the test data is changed

**Review by ZavezX — APPROVED** [review]

**Pull request merged**